modules/local/parse_kraken2report.nf

```diff
@@ -0,0 +1,180 @@
+process PARSE_KRAKEN2REPORT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::python=3.10.4"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/python:3.10.4' :
+        'biocontainers/python:3.10.4' }"
+
+    input:
+    tuple val(meta), path(kraken2report)
+
+    output:
+    tuple val(meta), path ("taxa_to_filter.txt"), emit: to_filter
+    path "versions.yml",                          emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    """
+    #!/usr/bin/env python
```

Comment:
this script is massive... how specific is this to detaxizer? maybe this can be a shared module, feels like other people might want to do something similar? or at least put it into `/bin`?